ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -920,10 +944,10 @@ def check_params(module, params, mode):
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            cmd=dict(required=True, choices=['create', 'bind', 'renumber', 'unbind', 'destroy', "connect-vms", "disconnect-vms"]),
+            cmd=dict(required=True, choices=['create', 'bind', 'bind_keysight_api_server_ip', 'renumber', 'unbind', 'destroy', "connect-vms", "disconnect-vms"]),
             vm_set_name=dict(required=False, type='str'),
             topo=dict(required=False, type='dict'),
-            vm_names=dict(required=True, type='list'),
+            vm_names=dict(required=False, type='list'),
```

Comment:
This has to be True. We cannot change the existing behavior. Why not pass an empty list from the caller?

Reply:
I think this is changed to work with Ixia stand alone testbed where no VM was defined/needed.

Reply:
The old design has been restored. For Keysight API Server there is no VM. An empty vm_names list is good to handle this scenario.